obelisk_ws/src/obelisk_msgs/obelisk_sensor_msgs/CMakeLists.txt

```diff
@@ -12,12 +12,15 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(obelisk_std_msgs REQUIRED)
```

Comment:
by having the messages rely on other messages, building them might be a bit of a pain. maybe we should add another pixi task that builds first the "core" messages that don't rely on other messages, then builds the other messages. Then we can make `messages-build` depend on this new task so everything is built correctly.